### DISCUSSION TITLE
新版动态中间变得很窄是什么原因

### DISCUSSION THREAD

**Discussion by yongrenx** [the1812/Bilibili-Evolved#4278] [提问 / Q&A]
![image](https://github.com/the1812/Bilibili-Evolved/assets/120461815/65ce13e1-4e05-43cc-8ee8-3ddc0ff0c5e3)
![image](https://github.com/the1812/Bilibili-Evolved/assets/120461815/f208e3b0-4425-4fdd-ae62-50b7a4fa5920)

**Comment by the1812** [comment]
因为目前没有很好的办法区分新旧版动态, 所以 `强制固定动态侧栏` 会按照旧版动态的尺寸进行设置

**Reply by yongrenx** [reply]
强制固定顶栏和侧栏的功能能否分开呢，我是只用到了固定顶栏的功能

**Reply by the1812** [reply]
如果使用自定义顶栏，那里有选项可以单独固定

**Reply by yongrenx** [reply]
那个在视频页也会固定，看来目前没什么好的解决办法，先这样用吧

**Reply by Geeyun-JY3** [reply]
破站自己的新版顶栏目前就已经是全局固定的了。

**Reply by yongrenx** [reply]
就是不想用破站的才美化嘛

**Reply by Geeyun-JY3** [reply]
所以你的需求是：使用`自定义顶栏`，并只在动态首页固定`自定义顶栏`？

**Reply by yongrenx** [reply]
是这样的

**Reply by the1812** [reply]
那可以去搞个 Stylus 样式,  设置为仅匹配动态首页的 URL
```css
.custom-navbar {
  position: fixed !important;
}
